orbital4c/orbital.py

```diff
@@ -40,6 +40,25 @@ def __sub__(self, other):
         output.comp_array = self.comp_array - other.comp_array
         return output
 
+    def save(self):
```

Comment:
You need to add a name here. Now we have only one orbital. Later on we will have several.